src/OSPSuite.R/Api.cs

```diff
@@ -0,0 +1,18 @@
+﻿using OSPSuite.R.Bootstrap;
+
+namespace OSPSuite.R
+{
+   public class ApiConfig
+   {
+      public string DimensionFilePath { get; set; }
+      public string PKParametersFilePath { get; set; }
+   }
+
+   public static class Api
```

Comment:
@Yuri05 Entry point for the R Interface. Need to be done once to bootstrap everything. Let's discuss naming later. We could call it Bootstrap or whatever